mockfaster/src/main/java/com/decathlon/tzatziki/matchers/StrictArrayContentJsonStringMatcher.java

```diff
@@ -31,7 +30,7 @@
  */
 public class StrictArrayContentJsonStringMatcher extends BodyMatcher<String> {
     private static final String[] EXCLUDED_FIELDS = {"mockServerLogger"};
-    private static final ObjectWriter PRETTY_PRINTER = ObjectMapperFactory.createObjectMapper(true);
+    private static final ObjectWriter PRETTY_PRINTER = ObjectMapperFactory.createObjectMapper().writer();
```

Comment:
Why removing pretty boolean ?

Reply:
The signature of the method has changed, you must now use the prettyprinter writer. I replace it in the code